.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/stale@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: >
+            Thank you for your contribution! To avoid pull requests to pile up,
+            an automated process marked this pull request as stale.
+            It will close this pull request if no further activity occurs.
```

Comment:
IMO, It would be nice to warn the contributor about the number of days of inactivity allowed. Thanks!

Reply:
How important is this for you? I thought about this, too. The problem I identified is that changes to the numbers will cause inconsistencies. If state PRs were already reported, their comment might be out of sync (say we lower the number to 7 instead of 14 days. The comment will still say 14 but the workflow will close it after 7). Also we might forget to update the number in the message since there is no indication that we can use a template for this number. Also contributors could re-open/re-file a new PR anytime anyhow (maybe we should add a message about this somehow, but it would belong in a close statement. What are you comments?

Reply:
It is not so important for me. Maybe we can document the number in CONTRIBUTING.md, this way if we change the number we are also going to change it in the documentation. Is that fins with you? If not, you can merge this anyway, is not that important for me :-)

Reply:
I would set a number for the visitor to be fair. Or at least a link to where this number is specified.

Regarding updates of the number, perhaps you can comment near the number that it needs to be updated in another place.
You can even set a test that makes sure the numbers are in sync.

Reply:
Adding a link to the config file works for me.